### ISSUE TITLE
Bookmark Groups and branches, enhance bookmark dialog

### ISSUE THREAD

**Issue by kamizoli** [mbnuqw/sidebery#220]
## Description
I only started using Sidebery today but like it very much already!
I would like to suggest a context menu item to bookmark a branche of tabs from the tree or the tabs of a group into a bookmark folder.  At the moment one has to select all the tabs of a branche, but it would be nice to have an option to bookmark a branche by selecting the root tab of this branche, like tree style tabs allows. 
Also the bookmark dialog should have an easier option to create a folder, than needing to create it via the context menu.
One possibiity would be when selecting multiple tabs to bookmark or a branche the dialog offers to create a folder straight away.

**Comment by mbnuqw** [comment]
Right now you can bookmark the branch of tabs with preserved tree state:
- Right click on the favicon of parent tab to select whole branch
- Bookmark

![Peek 2020-04-22 09-43](https://user-images.githubusercontent.com/6276694/79941454-c76fe900-847d-11ea-9806-646acf4f6307.gif)

Or did you mean something else?

**Comment by kamizoli** [comment]
You are right! Somehow I didn't look propperly and confused the the bookmark folder with the bookmarks themself.
The process would be helped, when after pressing the star to bookmark, a dialog would open where one could define the folder name an select a folder to save in. Like Firefox does, when one bookmarks tabs.
Also, at the moment, one can choose a folder to save in, but one has to mouse click the "ok", it would be nice if the "ok" would be preselected and one coud just hit enter to finish the dialog.

I think the "ok" should always be preselected in dialogs apart from confirming deletions and other harmful cases.

I have another off topic question. When I first saw the groups feature, with the page listing the tabs, I tought it would be like the one-tab extension, that the tabs where disolved into the group page. But the tabs are still there, when the group is unfolded. 
Of course one can discard the group to save memory, but at the moment there does not seem to be a big difference between groups and tree branches otherwise. Why did you choose it this way?
But maybe I have still to find out the best ways to use this feature, I'm still new to sidebery.

**Comment by mbnuqw** [comment]
>The process would be helped, when after pressing the star to bookmark, a dialog would open where one could define the folder name an select a folder to save in. Like Firefox does, when one bookmarks tabs.

You cat set the "Ask where to store bookmarks" option in the Tabs section of settings. This dialog though only allow you to set target directory at the moment.

---
>Also, at the moment, one can choose a folder to save in, but one has to mouse click the "ok", it would be nice if the "ok" would be preselected and one coud just hit enter to finish the dialog.

Sounds reasonable, especially for the keyboard guys. This will be implemented for sure.

---
Group page was added to manually group tabs in the tree structure (e.g. to separate different topics), so you are completely right - there is no difference.

**Comment by kamizoli** [comment]
I have activated the "Ask where to store bookmarks" setting. But still I would really appreciate it, if the process could be extended. Like #226 asks for.
